api/v0/cells/cells.proto

```diff
@@ -143,6 +145,17 @@ message CellServiceStopRequest {
 
 message CellServiceStopResponse {}
 
+message CellServiceListRequest {}
+
+message CellServiceListResponse {
+  repeated CellWithChildren cells = 1;
+}
+
+message CellWithChildren {
+  Cell cell = 1;
```

Comment:
I think we need to consider what data we want back. It seems weird to me to differ on what we want back in Allocate versus List. I don't think you should worry about it in this PR though, we can take care of it in a follow up PR.

@krisnova